src/components/Project/Project.js

```diff
@@ -191,6 +241,70 @@ const Project = ({
     [handleSetProjectData]
   )
 
+  const handleAddProjectLabel = (label, labels) => {
+    const objectLabels = {}
+
+    labels.forEach(label => {
+      const splitedLabel = label.split(':')
+      objectLabels[splitedLabel[0]] = splitedLabel[1].replace(' ', '')
```

Comment:
Just to make sure you do here what you expected to do:
Calling `String.prototype.replace` with a string value as the first argument will replace only the first occurrence of the sub-string. For example:
```js
'a b c d'.replace(' ', '')
// returns "ab c d"
```
You can pass a RegExp value with the global flag, `g`, and that will replace all occurrences:
```js
'a b c d'.replace(/ /g, '')
// returns "abcd"
```
Instead of using the space character, you can also use the RegExp character class `\s`, which matches whitespace characters, such as spaces, tabs, & line breaks. For example:
```js
'a b c d'.replace(/\s/g, '')
// returns "abcd"
```
Since ES2021 we also have `String.prototype.replaceAll`, which lets us replace all occurrences by passing a string value and not a RegExp value, for example:
```js
'a b c d'.replaceAll(' ', '')
// returns "abcd"
```
This is the preferred approach since it is more readable than RegExp.

But before you switch to use any of the above, the real question here is: did you intend to replace only the first space in `splitedLabel[1]` or all the spaces. If all, use `replaceAll` instead of `replace`.